recipe/build-mpi.sh

```diff
@@ -1,27 +1,21 @@
 #!/bin/bash
 
-# configure balks if F90 is defined
-# with a fatal deprecation message pointing to FC
+# unset unused old fortran compiler vars
```

Comment:
The previous comment was more informative. Is there a a reason to delete this context?

Reply:
Probably not. Let's revert that.